client/blocks/support-article-dialog/dialog.jsx

```diff
@@ -122,7 +131,10 @@ const getPostKey = memoize(
 );
 
 const mapStateToProps = ( state ) => {
-	const postId = getInlineSupportArticlePostId( state );
+	let postId = getInlineSupportArticlePostId( state );
+	if ( ! postId ) {
+		postId = parseInt( getUrlSearchQuery( 'support-article' ) );
```

Comment:
When using `parseInt()` don't forget to add the second `radix` argument. 

You can read more about why in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
Good call.